### PR TITLE
CIV-12408 DRH SDO

### PR DESCRIPTION
**Pull request by bhagyashreesharma90**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-12408

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/constants/SdoR2UiConstantSmallClaim.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        b) A copy of the credit agreement;
+        c) A copy of the PPI policy application and agreement;
+        d) Any cancellation notices;
+        e) Any correspondence in connection with a complaint by the Claimant(s);
+        f) A copy of any correspondence as to redress if not included in (e);
+        g) The amount, if any, of redress (under the FCA scheme or otherwise) and the date it was paid.
+
+        The parties shall endeavour to agree a calculation of the sums in issue and upload to the Digital Portal the agreed calculation.\
+         In default of agreement each party shall upload to the Digital Portal its own calculation not later than 7 days before any hearing.\
+         The calculation must show the amount which is intended to remove any unfairness from the relationship,broken down into:
```
Comment (hmcts-version1-ignacio) [inline src/main/java/uk/gov/hmcts/reform/civil/constants/SdoR2UiConstantSmallClaim.java:27]: A space after the comma is missing. 
"relationship,broken"

**Review by hmcts-version1-ignacio — COMMENTED** [review]
Just one typo to check

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java [hunk 2]
```diff
@@ -666,6 +685,9 @@ private CallbackResponse prePopulateOrderDetailsPages(CallbackParams callbackPar
 
         updatedData.smallClaimsRoadTrafficAccident(tempSmallClaimsRoadTrafficAccident).build();
 
+        if (featureToggleService.isSdoR2Enabled()) {
+            populateDRHFields(callbackParams, updatedData, preferredCourt);
```
Comment (ShwetaTandel-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java:689]: Should we not check the DRH is selected along with sdo r2 flag

**Review by ShwetaTandel-hmcts — COMMENTED** [review]
add comments

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java (hunk 2 above)
Comment (bhagyashreesharma90) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java:689]: > Should we not check the DRH is selected along with sdo r2 flag

No.. This is in about to start so we don't have if user is selected DRH or not.

File: src/main/java/uk/gov/hmcts/reform/civil/constants/SdoR2UiConstantSmallClaim.java (hunk 1 above)
Comment (bhagyashreesharma90) [reply src/main/java/uk/gov/hmcts/reform/civil/constants/SdoR2UiConstantSmallClaim.java]: > A space after the comma is missing. "relationship,broken"

Done

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Pull request merged**